### PR TITLE
[ACS-4124] Display tags applied to file in files list

### PR DESCRIPTION
**Pull request by MichalKinas**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

Tags applied to a file are not displayed.

**What is the new behaviour?**

Tags applied to a file are displayed on the file list. https://alfresco.atlassian.net/browse/ACS-4124

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    class: 'adf-datatable-content-cell aca-tags-name-column'
+  }
+})
+export class TagsColumnComponent {
+  @Input()
+  context: any;
+
+  constructor(private cd: ChangeDetectorRef) {}
+
+  getNodeId({ row }): string {
```
Comment (jatin2008) [inline projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:45]: do we require test case for the new method added?

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    return row.id;
+  }
+
+  onTagsLoaded(): void {
```
Comment (jatin2008) [inline projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:49]: same for this method do we require test case as new code is being added

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts (hunk 1 above)
Comment (AleksanderSklorz) [inline projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:45]: Can we set type for row?

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  `,
+  styleUrls: ['./tags-column.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  host: {
+    class: 'adf-datatable-content-cell aca-tags-name-column'
+  }
+})
+export class TagsColumnComponent {
+  @Input()
+  context: any;
```
Comment (AleksanderSklorz) [inline projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:41]: Is any required? Don't we have some type to use it here?

Comment (MichalKinas) [reply projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:41]: I don't think we have some type to assign here, as far as I checked for each custom column any is used as context, even `DataColumnComponent` in ADF has `customData` set to any. In ACA `SearchResultsRowComponent` might serve as example

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts (hunk 2 above)
Comment (MichalKinas) [reply projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:49]: Same here, this method is just used to trigger change detection without any logic inside @AleksanderSklorz what do you think?

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts (hunk 1 above)
Comment (MichalKinas) [reply projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:45]: I changed it a bit so we can use `ShareRowData` as type for row

Comment (AleksanderSklorz) [reply projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts]: yeah it is very simple function (it is just get in reality) so probably skipping tests for that won't be huge problem

File: projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts (hunk 2 above)
Comment (AleksanderSklorz) [reply projects/aca-content/src/lib/components/dl-custom-components/tags-column/tags-column.component.ts:51]: agreed

**Review by AleksanderSklorz — APPROVED** [review]

File: projects/aca-content/assets/app.extensions.json [hunk 4]
```diff
@@ -2076,6 +2076,17 @@
...
+        },
+        {
+          "id": "app.filer.tags",
+          "template": "app.columns.tags",
+          "key": "tags",
+          "title": "APP.DOCUMENT_LIST.COLUMNS.TAGS",
+          "class": "adf-full-width adf-expand-cell-4",
+          "type": "text",
+          "sortable": false,
+          "desktopOnly": false,
```
Comment (DenysVuika) [inline projects/aca-content/assets/app.extensions.json:2088]: you may want to put true for smaller screens.... what it does - hides the column on the small screens

**Review by DenysVuika — APPROVED** [review]

File: projects/aca-content/assets/app.extensions.json (hunk 4 above)
Comment (MichalKinas) [reply projects/aca-content/assets/app.extensions.json:2088]: Yeah I thought about it as well, I'll consult it with Lilian and Chapman to be sure if we want to hide it

Comment (MichalKinas) [reply projects/aca-content/assets/app.extensions.json]: Lilian confirmed that we want to hide tags for smaller displays, changed to `true` now

**Pull request merged**